traffic_ops/testing/compare/compare.go

```diff
@@ -266,11 +266,11 @@ func handlePlainTextResponse(responses *[]result, route string) {
 		log.Printf("Identical results (%d bytes) from %s\n", len(result0), route)
 	} else {
 		writeAllResults(route,
-		                result0Str,
-		                (*responses)[0].TO,
-		                result1Str,
-		                (*responses)[1].TO,
-		                checkOrderDiffs(scrubbedResult0, scrubbedResult1))
+			result0Str,
+			(*responses)[0].TO,
+			result1Str,
+			(*responses)[1].TO,
+			checkOrderDiffs(scrubbedResult0, scrubbedResult1))
```

Comment:
alignment is bad?

Reply:
according to gofmt :)

Reply:
I really hate that, but oh well I guess.

Reply:
Consider the following gofmt friendly layout:

```
writeAllResults(
    route,
    ...
    checkOrderDiffs(...),
)
```

Reply:
Yeah I've used that before. Still prefer the original.